src/App.js

```diff
@@ -1,28 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Calculator />
+    </>
```

Comment:
- [ ] kindly make your calculator to match the given design, your `=` should be of same background color with other operators 
your design :point_down: 
![Screenshot from 2023-02-25 11-34-23](https://user-images.githubusercontent.com/66097371/221352376-b9826f5d-66cc-429e-ae91-f763e13f2160.png)

design given :point_down: 
![Screenshot from 2023-02-25 11-37-57](https://user-images.githubusercontent.com/66097371/221352431-0f97f212-5bbd-4b4d-9fca-dbc268d840bf.png)

Reply:
- Hey, As requested by the previous reviewer [here](https://github.com/kazaneza/calculator/pull/6#discussion_r1117901392), please, kindly update your calculator to match the wireframe to improve your design.